src/core/core.scale.js

```diff
@@ -569,6 +569,60 @@ module.exports = function(Chart) {
 				0;
 		},
 
+		/**
+		 * Returns a subset of ticks to be plotted.
+		 * This functionality makes it so that we don't have overlapping labels.
+		 */
+		autoSkip: function() {
```

Comment:
I'm not a big fan of `autoSkip`, but actually was thinking of `_decimate` (not `_disseminate`). We should make it private until there is a need/request for it to be public, we don't want to maintain this kind of backward compatibility.